ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/ReplaceWithPredefinedOperatorFix.fs

```diff
@@ -0,0 +1,37 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Resources.Shell
+
+type ReplaceWithPredefinedOperatorFix(error: AddingConstraintError) =
+    inherit FSharpQuickFixBase()
+
+    let ref = error.Expr.As<IReferenceExpr>()
+
+    let fsharpOperator =
+        match ref with
+        | null -> SharedImplUtil.MISSING_DECLARATION_NAME
+        | ref when ref.IsQualified -> SharedImplUtil.MISSING_DECLARATION_NAME
```

Comment:
So if there's a name and a qualifier then you'll say that name is missing?

Reply:
I think this line is not needed at all, since qualified access to the operator presupposes a prefix notation